modules/dr-restoring-etcd-quorum-ha.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * disaster_recovery/quorum-restoration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="dr-restoring-etcd-quorum-ha_{context}"]
+= Restoring etcd quorum for high availability clusters
+
+You can use the `quorum-restore.sh` script to instantly brings back a new single-member etcd cluster based on its local data directory and marks all other members as invalid by retiring the previous cluster identifier. No prior backup is required to restore the control plane from.
```

Comment:
to instantly _bring_ back [...] and _mark_ all other members as invalid

Reply:
Typo, thanks!